tests/Command/ImportMappingDoctrineCommandTest.php

```diff
@@ -66,6 +66,7 @@ protected function tearDown(): void
         unset($this->kernel, $this->commandTester);
     }
 
+    #[WithoutErrorHandler]
```

Comment:
The proper way to write tests for legacy APIs is not to disable the error handler (which will also not report notices and warnings) but to add the `IgnoreDeprecations` attribute, which disables only the reporting of deprecations.